libs/ui/select/select.stories.ts

```diff
@@ -532,11 +627,7 @@ export const CustomTrigger: Story = {
 	`,
 })
 export class CustomSelectTriggerComponent {
-	@ViewChild('button', { static: true })
-	public buttonEl!: ElementRef;
-
-	@ContentChild(HlmIconComponent, { static: false })
-	protected icon!: HlmIconComponent;
+	protected icon = contentChild(HlmIconComponent);
```

Comment:
should probably be readonly. we might want to extend our linting to take care of that also